src/widget/wbeatspinbox.cpp

```diff
@@ -41,7 +41,13 @@ void WBeatSpinBox::stepBy(int steps) {
     }
     // Do not call QDoubleSpinBox::setValue directly in case
     // the new value of the ControlObject needs to be confirmed.
+    // Curiously, m_valueControl.set() does not cause slotControlValueChanged
+    // to execute for beatjump_size, so call setValue() directly here.
+    double oldValue = m_valueControl.get();
     m_valueControl.set(newValue);
```

Comment:
I don't understand why this causes slotControlValueChanged to execute for beatloop_size but not beatjump_size. beatloop_size requires confirmation before setting a new value but beatjump_size does not.